content/docs/user-guide/gems/reference/multiplayer/multiplayer/tutorial_multiplayer.md

```diff
@@ -0,0 +1,887 @@
+        : MyFirstNetworkComponentControllerBase( parent )
+    {
+    }
+
+    void MyFirstNetworkComponentController::OnActivate( [[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating )
+    {
+        AZ::TickBus::Handler::BusConnect();
+    }
+
+    void MyFirstNetworkComponentController::OnDeactivate( [[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating )
+    {
+        AZ::TickBus::Handler::BusDisconnect();
+    }
+
+    void MyFirstNetworkComponentController::OnTick( [[maybe_unused]] float deltaTime, AZ::ScriptTimePoint time )
+    {
+        ModifyUpTime() = time.GetSeconds();
+        AZ_Printf( "MyFirstNetworkComponent", "server = %f", GetUpTime() );
+    }
+}
+```
+
+Our code gen xml file `D:\git\o3de-multiplayersample\Gem\Code\Source\AutoGen\MyFirstNetworkComponent.AutoComponent.xml` is:
+
+```xml
+<?xml version="1.0"?>
+<Component Name="MyFirstNetworkComponent"
+           Namespace="MultiplayerSample"
+           OverrideComponent="true"
+           OverrideController="true"
+           OverrideInclude="Source/Components/MyFirstNetworkComponent.h"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <NetworkProperty Type="double"
+                   Name="UpTime"
+                   Init="0.0"
+                   ReplicateFrom="Authority"
+                   ReplicateTo="Client"
+                   IsRewindable="true"
+                   IsPredictable="true"
+                   IsPublic="true"
+                   Container="Object"
+                   ExposeToEditor="false"
+                   ExposeToScript="false"
+                   GenerateEventBindings="true"
+                   Description="Time since the start of the application" />
+</Component>
+```
+
+And now we have a clean separation between server and client logic for our component.
+
+### Add a Remote Procedure Call
+
+Now that we have created a data flow from the server to clients, let's reverse the direction with a remote procedure call. Building from the previous example, whenever a client gets new `UpTime` it will call a new RPC - `SendConfirmUptime` and send the value it got back to the server. Just for fun.
+
+```xml
+  <RemoteProcedure Name="SendConfirmUptime"
+                   InvokeFrom="Autonomous"
+                   HandleOn="Authority"
+                   IsPublic="false"
+                   IsReliable="false"
+                   GenerateEventBindings="false"
+                   Description="Uptime confirmed by the client">
+    <Param Type="double"
+           Name="UpTime" />
+  </RemoteProcedure>
+```
+
+{{<important>}}
+Notice that InvokeFrom is not `Client` but `Autonomous`. `Autonomous` is a special type of client behavior where a client initiates an action and may send data to the server. A common example is a player character controller and actions that has to be able to act on its own. Meanwhile, 'Client' only mirrors what the server tells them to do.
```

Comment:
'Autonomous' is commonly used for player character controllers; in this case the player has to be able to act on its own without waiting for the server to tell it what to do.

Reply:
Sounds good, fixing.